apps/codeforafrica/src/payload/globals/Site/index.js

```diff
@@ -2,8 +2,9 @@ import EngagementTab from "./EngagementTab";
 import GeneralTab from "./GeneralTab";
 import NavigationTab from "./NavigationTab";
 
-const Settings = {
-  slug: "settings",
+const Site = {
+  slug: "site",
```

Comment:
@kilemensi Not sure about it, but since we already have some data in prod, do we need to update this slug, or should we keep it as it was: `settings`?

Reply:
Yes, let's update it @kelvinkipruto. I don't think the content team has had time to really dig in just yet. Keep a copy of the JSON data handy before deploying & we can update the data after deployment.

Reply:
For all the above changes, we can use slug as is i.e. `site`, `publication`, etc., or prepend `setting-` to show these are settings related i.e. `settings-site`, `settings-publication`, etc.

I'll let you be the architect.